### ISSUE TITLE
Unable to set pool member attributes

### ISSUE THREAD

**Issue by pteiber** [f5devcentral/terraform-provider-bigip-old#74]
The provider doesn't allow for pool member-specific attributes to be set (like priority group, connection limit,  member-specific health monitors, state, or session).

As a basic example, I'd like to replicate this LTM configuration:
```
ltm pool test-pool {
    members {
        10.0.0.100:http {
            address 10.0.0.100
            priority-group 100
        }
        10.0.0.200:http {
            address 10.0.0.200
            priority-group 200
        }
    }
}
```

I've already been experimenting with using a nested resource to add this functionality, but so far everything I have tried has become complicated very quickly because of the existing `nodes` attribute in the bigip_ltm_pool resource.

**Comment by jaloren** [comment]
I have also encountered this problem because i wanted to provision priority group for an f5 pool. Since the provider is usging go-bigip rest api client, I checked to see if this can be done and looks like it can.

https://godoc.org/github.com/f5devcentral/go-bigip#BigIP.ModifyPoolMember

Does f5 see value you in adding this functionality? Would they accept a PR from an external contributor to add it in? I haven't done a deep dive yet, I don't think it would be super complicated.

**Comment by dannyk81** [comment]
Hi @jaloren, I recently proposed a new pool attachment resource (`bigip_ltm_pool_attachment`) which was merged (https://github.com/f5devcentral/terraform-provider-bigip/pull/93) to master.

This new resource is now handling the node membership in pool(s), and the `nodes` attribute was removed from `bigip_ltm_pool`, so the proposed member configuration can be easily added to the new resource.

Although we may need to use `CreatePoolMember` instead of `AddPoolMember`.
